<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InteriorLights</class>
 <widget class="QDialog" name="InteriorLights">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(22, 42, 171);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>281</width>
     <height>162</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="frontLights">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(41, 245, 255);</string>
      </property>
      <property name="text">
       <string>Front Lights</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="midLights">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(41, 245, 255);</string>
      </property>
      <property name="text">
       <string>Mid Lights</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="backLights">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(41, 245, 255);</string>
      </property>
      <property name="text">
       <string>Back Lights</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="backFIL">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
background-color: rgb(138, 195, 255);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
